### ISSUE TITLE
Response from VertexAIModelGarden is unexpected.

### ISSUE THREAD

**Issue by marshmellow77** [langchain-ai/langchain-google#72]
Model response from VertexAIModelGarden is unexpected. It is in one of two formats:

```
Prompt
<prompt>
Output:
<prompt>
<model_response>
```

<img width="930" alt="image" src="https://github.com/langchain-ai/langchain-google/assets/63367770/4d6d3121-108e-43de-ac57-8e965048d8ae">

or

```
Prompt
<prompt>
Output:
<model_response>
```

<img width="640" alt="image" src="https://github.com/langchain-ai/langchain-google/assets/63367770/878b67dd-b622-4db1-9da4-198cba34a549">

In either case this response format is unexpected for Langchain components and will break them (e.g. ReAct agents).

**Comment by marshmellow77** [comment]
I've submitted a PR that fixes the issue : #73

**Comment by pprados** [comment]
Why the PR is closed ?
It's impossible to use `VertexAIModelGarden` now.
